### DISCUSSION TITLE
Ecommerce events/actions for Shopify Basic (not Plus)

### DISCUSSION THREAD

**Discussion by abadger13** [merchantprotocol/theadslab#47] [User Guide Q&A]
Hi Jonathon,
My clients are not using Shopify Plus, they're all in Basic.
Therefore we can't install the cart tracking for the customer journey, ie the events like the FB pixel View Content, Add to Cart, Add Payment, and Purchase.

Is there a way to track actions of visitors through the site pages and through the purchase path?
Thanks,
Amy

EDIT: I think I found where to do this -- in the tracking code installation guide. "If you want to do more than track page views, please check out the Matomo documentation at https://developer.matomo.org/guides/tracking-javascript-guide.
You can also generate a customized tracking code at https://app.theadslab.io/index.php?module=CoreAdminHome&action=trackingCodeGenerator&idSite=337&period=day&date=2022-05-04."

I'm going to see if I can figure this out. Not a coder, just an innocent bystander.

**Comment by jonathonbyrdziak** [comment]
lol. well let me know if you were able to get it figured out on your own.

**Comment by abadger13** [comment]
Looks like the ecommerce tracking happens inside of Matomo and login has to be Super User.
https://matomo.org/faq/reports/set-up-ecommerce-tracking-in-matomo/
I didn't see any way to track ecommerce in the Ads Lab dashboard or tabs there.

So no, I wasn't able to figure it out on my own.
